doc/nrf/security/crypto/drivers.rst

```diff
@@ -344,10 +344,13 @@ CRACEN driver
    * - :ref:`CRACEN <ug_nrf54l_cryptography>`
      - Hardware
      - Open-source
-     - nRF54L Series devices
+     - nRF54L Series devices, nRF54H20
 
 The CRACEN driver provides entropy and hardware-accelerated cryptography using the Crypto Accelerator Engine (CRACEN) hardware peripheral.
-For more information about both the driver and the hardware peripheral, see :ref:`ug_nrf54l_crypto_kmu_cracen_peripherals` on the :ref:`ug_nrf54l_cryptography` page.
+The hardware peripheral is available on the following devices:
+
+* nRF54L15 Series devices - For more information about both the hardware peripheral and the driver, see the `CRACEN hardware peripheral <nRF54L15 DK CRACEN_>`_ page in the datasheet and the :ref:`ug_nrf54l_cryptography` page in the |NCS| documentation.
+* nRF54H20 - For more information, see the :ref:`ug_nrf54h20_secure_domain_cracen` and the IronSide Secure Element ``ug_nrf54h20_ironside_se`` pages.
 
 CRACEN driver configuration
 ---------------------------
```

Comment:
This comment is for psa_crypto_api_ironside.svg.

Oberon PSA Crypto and the driver wrapper exists inside the Secure Domain box only.

The orange box for "IronSide SE firmware bundle" should be within "Secure Domain".